<?xml version="1.0"?>
<launch>
  <arg name="map_topic" default="/map" />
  <arg name="map_file" default="$(find utexas_gdc)/maps/3ne-real-new.yaml" />
  <arg name="map_frame" default="map" />
  <arg name="map_service" default="/static_map" />
  <arg name="tf_prefix" default="" />
  <arg name="base_frame_id" default="base_footprint" />
  <arg name="odom_frame_id" default="odom" />

  <include file="$(arg robot_configuration)" />

  <!-- <node name="map_server" pkg="map_server" type="map_server" 
        args="$(arg map_file)" /> -->

  <include file="$(find vizzy_navigation)/launch/amcl.launch" />
  <include file="$(find vizzy_navigation)/launch/move_base_eband.launch">
    <arg name="map_topic" value="$(arg map_topic)" />
  </include>
  <include file="$(find vizzy_navigation)/launch/rviz.launch" />
  <node name="fake_localization" pkg="fake_localization" type="fake_localization">
    <remap from="base_pose_ground_truth" to="odom"/>
    <param name="global_frame_id" value="$(arg tf_prefix)/$(arg map_frame)" />
    <param name="base_frame_id" value="$(arg tf_prefix)/$(arg base_frame_id)" />
    <param name="odom_frame_id" value="$(arg tf_prefix)/$(arg odom_frame_id)" />
  </node>

</launch>
